modules/getting-started/proc-customize-rhdh-learning-paths.adoc

```diff
@@ -0,0 +1,85 @@
+====
+
+You can provide data to the Learning Path from the following sources:
+
+* JSON files hosted on GitHub or GitLab.
+* A dedicated service that provides the Learning Path data in JSON format using an API.
+
+== Using hosted JSON files to provide data to the Learning Paths
+
+.Prerequisites
+
+You have installed {product} by using either the Operator or Helm chart. For more information, see xref:proc-install-rhdh_rhdh-getting-started[].
+
+.Procedure
+
+To access the data from the JSON files, complete the following step:
+
+* Add the following code to the `app-config.yaml` file:
++
+[source,yaml]
+----
+proxy:
+  endpoints:
+    '/developer-hub':
+      target: https://raw.githubusercontent.com/
+      pathRewrite:
+        '^/api/proxy/developer-hub/learning-paths': '/janus-idp/backstage-showcase/main/packages/app/public/learning-paths/data.json'
+        '^/api/proxy/developer-hub/tech-radar': '/janus-idp/backstage-showcase/main/packages/app/public/tech-radar/data-default.json'
+        '^/api/proxy/developer-hub': '/janus-idp/backstage-showcase/main/packages/app/public/homepage/data.json'
+      changeOrigin: true
+      secure: true
+----
+
+== Using a dedicated service to provide data to the Learning Paths
+
+When using a dedicated service, you can do the following:
+
+* Use the same service to provide the data to all configurable {product-short} pages or use a different service for each page.
+* Use the https://github.com/redhat-developer/red-hat-developer-hub-customization-provider[`red-hat-developer-hub-customization-provider`] as an example service, which provides data for both the Home and Tech Radar pages. The `red-hat-developer-hub-customization-provider` service provides the same data as default {product-short} data. You can fork the `red-hat-developer-hub-customization-provider` service repository from GitHub and modify it with your own data, if required.
+* Deploy the `red-hat-developer-hub-customization-provider` service and the {product-short} Helm chart on the same cluster.
+
+.Prerequisites
+
+* You have installed the {product} using Helm chart. For more information, see xref:proc-install-rhdh_rhdh-getting-started[].
+
+.Procedure
+
+To use a dedicated service to provide the Learning Path data, complete the following steps:
+
+. Add the following code to the `app-config-rhdh.yaml` file:
++
+[source,yaml]
+----
+   proxy:
+      endpoints:
+        # Other Proxies
+        '/developer-hub/learning-paths':
+          target: ${LEARNING_PATH_DATA_URL}
+          changeOrigin: true
+          # Change to "false" in case of using self hosted cluster with a self-signed certificate
+          secure: true
+----
+where the `LEARNING_PATH_DATA_URL` is defined as `pass:c[http://<SERVICE_NAME>/learning-paths]`, for example, `pass:c[http://rhdh-customization-provider/learning-paths]`.
++
+[NOTE]
+====
```

Comment:
@linfraze Similar to previous common about avoiding admonition close to each other.